### PR TITLE
Initial branding

### PR DESCRIPTION
**Pull request by rossneilson**
# Description

Initial logos from the demo branch
Also a null check

Closes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [ ] My code follows the existing style of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any UI changes have been tested and made responsive for mobile views

### CODE REVIEW

**Comment by Filipv95** [comment]
Hey @rossneilson, I've checked the logo, how can I test null check?

**Comment by rossneilson** [comment]
It was just a crash that happened when opening token list (but it now wouldn't crash as before since we removed the global error boundary)

File: src/components/Header/Header.styled.ts [hunk 1]
```diff
@@ -35,3 +36,8 @@ export const MenuItems = styled(Box)`
 export const ClickableHeading = styled(Heading)`
   cursor: pointer;
 `;
+
+export const TextLogo = styled(DaviLogo)`
+  width: 40%;
```
Comment (mprasanjith) [inline src/components/Header/Header.styled.ts:41]: Using the width as a percentage here breaks the navbar on mobile. I think its good enough for now, but we should fix this later.

Comment (Kenny-Gin1) [reply src/components/Header/Header.styled.ts:41]: If you want to set it to a percentage also use a max width.

**Comment by Filipv95** [comment]
Test passed and everything looks good. Great job!

**Pull request merged**